js/src/common/utils/Drawer.js

```diff
@@ -31,7 +31,18 @@ export default class Drawer {
    * @public
    */
   hide() {
-    $('#app').removeClass('drawerOpen');
+    const $app = $('#app');
+
+    // If the drawer isn't open, stop execution
```

Comment:
This comment is probably unnecessary.

Also, I think it'd be nice to have a comment at the top explaining the "why": This ensures that drawer elements aren't keyboard selectable while off screen.